src/links.js

```diff
@@ -2,8 +2,8 @@ export default [
   { href: '/', text: 'Home' },
   { href: '#about', text: 'About' },
   { href: 'https://www.freecodecamp.org/', text: 'FreeCodeCamp' },
-  { href: 'https://discord.gg', text: 'Discord' },
+  { href: 'https://discord.gg/aF8skEe9nq', text: 'Discord' },
```

Comment:
Good call, I didn't even know it would link back to my account. I did set it to never expire, however I agree we should have the invited come from the admin account.

Reply:
<img width="575" alt="Screenshot 2023-08-02 at 11 48 43 AM" src="https://github.com/FCCColumbus/cbus-web/assets/22714448/5b9193d4-ded9-4379-9840-ee4b0b60d9b6">

Uploading a photo to verify that the new Discord link is one generated by John. Resolving convo.